src/scripts.js

```diff
@@ -1,15 +1,54 @@
 import './styles.css';
 import RecipeRepository from './classes/RecipeRepository'
-import { fetchApiData } from './apiCalls';
 import burger from './images/burger.png';
 import './images/favorite.svg';
 import './images/fridge.svg';
 import './images/home.svg';
 import './images/list.svg';
 import './images/search.svg';
+import { fetchUsers, fetchIngredients, fetchRecipes} from './apiCalls.js'
+import IngredientsLibrary from './classes/IngredientsRepository.js'
+import Recipe from './classes/Recipe.js'
+import RecipeRepository from './classes/RecipeRepository'
+
+
+
+//global variables, instantiations of classes, holds info
+let ingredients, recipeRepository, recipe 
+
+
+const fetchData = () => {
+  Promise.all([fetchUsers(), fetchIngredients(), fetchRecipes()])
+    .then(data => parseData(data))
+}
+
+const parseData = (data) => {
+  let userDataArray = data[0].usersData
+  let ingredientDataArray = data[1].ingredientsData
+  let recipeDataArray = data[2].recipeData
+
+  instantiation(userDataArray, ingredientDataArray, recipeDataArray)
+  renderinfo()
+}
 
-console.log('Hello world');
+const instantiation = (userDataArray, ingredientDataArray, recipeDataArray) => {
+  //math random for random user, do not have user class yet
+  let i = Math.floor(Math.random()*42);
```

Comment:
could we use let i = Math.floor(Math.random()*userDataArray.length)    rather than the 42?